### ISSUE TITLE
estimates.R scale is different from estimates.YrA

### ISSUE THREAD

**Issue by agiovann** [flatironinstitute/CaImAn#826]
For better support, please use the template below to submit your issue. When your issue gets resolved please remember to close it.

Sometimes errors while running CNMF occur during parallel processing which prevents the log to provide a meaningful error message. Please reproduce your error with setting `dview=None`.

If you need to upgrade CaImAn follow the instructions given in the [documentation](https://caiman.readthedocs.io/en/master/Installation.html#upgrading).

- **Tell us a bit about your setup**: 
 1. Operating system (Linux/macOS/Windows): 
 macOS
 2. Python version (3.x): 
 3.7.1
 3. Working environment (Python IDE/Jupyter Notebook/other):
 Spyder
 4. Which of the demo scripts you're using for your analysis (if applicable):
 demo_pipeline.py
 5. CaImAn version*: 
 dev branch
 6. CaImAn installation process (`pip install .`/`pip install -e .`/conda): 
pip install -e .
*You can get the CaImAn version by creating a `params` object and then typing `params.data['caiman_version']`. If the field doesn't exist, type N/A and consider upgrading)

- **Describe the issue that you are experiencing**

When plotting using the command 
```cnm2.estimates.view_components(images, img=Cn,
                                      idx=cnm2.estimates.idx_components)```

the plotting is off by several orders of magnitude. I identified the problem is some normalization of estimates.R. estimates.YrA still works as it should, but the scale of estimates.R seems off. Not sure what introduced the issue.  

- **Copy error log below**



- **If you're not reporting an error, type your message below**

**Comment by j-friedrich** [comment]
Fixed in the dev branch (PR #827)

**Comment by agiovann** [comment]
Thank you Johannes!
